File: OPENAPI-REST-API/swagger-client/dart/lib/model/inline_response20020.dart
part of swagger.api;

class InlineResponse20020 {
  
  bool continue_ = null;

  List<String> errors = [];

  int serviceType = null;

  double serviceCost = null;

  double originalCost = null;

  double repeatServiceCost = null;

  InlineResponse20020();

  @override
  String toString() {
    return 'InlineResponse20020[continue_=$continue_, errors=$errors, serviceType=$serviceType, serviceCost=$serviceCost, originalCost=$originalCost, repeatServiceCost=$repeatServiceCost, ]';
  }

  InlineResponse20020.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    continue_ = json['continue'];
    errors = (json['errors'] as List).map((item) => item as String).toList();
    serviceType = json['serviceType'];
    serviceCost = json['serviceCost'];
    originalCost = json['originalCost'];
    repeatServiceCost = json['repeatServiceCost'];
  }

  Map<String, dynamic> toJson() {
    return {
      'continue': continue_,
      'errors': errors,
      'serviceType': serviceType,
      'serviceCost': serviceCost,
      'originalCost': originalCost,
      'repeatServiceCost': repeatServiceCost
     };
  }

  static List<InlineResponse20020> listFromJson(List<dynamic> json) {
    return json == null ? new List<InlineResponse20020>() : json.map((value) => new InlineResponse20020.fromJson(value)).toList();
  }

  static Map<String, InlineResponse20020> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, InlineResponse20020>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new InlineResponse20020.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/InlineResponse20020.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class InlineResponse20020 {
    /// <summary>
    /// Gets or Sets _Continue
    /// </summary>
    [DataMember(Name="continue", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "continue")]
    public bool? _Continue { get; set; }

    /// <summary>
    /// Gets or Sets Errors
    /// </summary>
    [DataMember(Name="errors", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "errors")]
    public List<string> Errors { get; set; }

    /// <summary>
    /// Gets or Sets ServiceType
    /// </summary>
    [DataMember(Name="serviceType", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "serviceType")]
    public int? ServiceType { get; set; }

    /// <summary>
    /// Gets or Sets ServiceCost
    /// </summary>
    [DataMember(Name="serviceCost", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "serviceCost")]
    public decimal? ServiceCost { get; set; }

    /// <summary>
    /// Gets or Sets OriginalCost
    /// </summary>
    [DataMember(Name="originalCost", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "originalCost")]
    public decimal? OriginalCost { get; set; }

    /// <summary>
    /// Gets or Sets RepeatServiceCost
    /// </summary>
    [DataMember(Name="repeatServiceCost", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "repeatServiceCost")]
    public decimal? RepeatServiceCost { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class InlineResponse20020 {\n");
      sb.Append("  _Continue: ").Append(_Continue).Append("\n");
      sb.Append("  Errors: ").Append(Errors).Append("\n");
      sb.Append("  ServiceType: ").Append(ServiceType).Append("\n");
      sb.Append("  ServiceCost: ").Append(ServiceCost).Append("\n");
      sb.Append("  OriginalCost: ").Append(OriginalCost).Append("\n");
      sb.Append("  RepeatServiceCost: ").Append(RepeatServiceCost).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
